addons/html_builder/static/src/plugins/carousel_option_plugin.js

```diff
@@ -51,6 +51,11 @@ export class CarouselOptionPlugin extends Plugin {
         on_snippet_dropped_handlers: this.onSnippetDropped.bind(this),
         normalize_handlers: this.normalize.bind(this),
         on_reorder_items_handlers: this.reorderCarouselItems.bind(this),
+        clean_for_save_handlers: ({ root }) => {
+            for (const carouselEl of root.querySelectorAll(".carousel")) {
+                this.slide(carouselEl, 0);
+            }
+        },
```

Comment:
I m not sure that is a good idea to call slide. Slide is async, it will do a setTimeout to apply the active class

Reply:
I dont know if we just need to move the active class to the first slide or if we need to update some other stuff with bootstrap

Reply:
I meant we don't care much cause when we click save, there's a gray semi-transparent overlay on the whole website editing page. I think the overlay already giving the info that the website is being modified and saved. :thinking: 

I think it's more a PO decision so I'll ping @lebl-odoo , the question is: shall we have the sliding effect when setting the carousel back to the first slide on save? :pray: